pandunia-loge.csv

```diff
@@ -1034,6 +1034,7 @@ gabr.o.xek:e|gabroxeke||||tombstone (gravestone)|pierre tombale|lápida|lápide|
 gad:e|gade|biv|Equus africanu asinus|hin:गधा (gadhā), ben:গাধা (gadha), guj:ગધેડો (gadheḍo), tel:(gāḍida), tam:(kaḻutai), may:keledai, + yor:ké̩té̩ké̩té̩|donkey (ass)||burro (asno)|||||||||||||azeno|aasi|osioł
 gad.o.gud:e|gadogude|biv|Equus asinus♂ × Equus caballus♀||mule|mule|mula||мул||骡|騾馬|||||||||muuli|muł
 gadolin.em:|gadolinem|mate|Gd||gadolinium||gadolinio|||||||||||||gadolinio|gadolinium|gadolin
+gaf:|gaf|biv||eng:woof, spa:guau, rus:гав (gav), tur:hav, zho:汪 (wāng), jpn:ワン (wan)|dog||perro||||||||||||||koira|pies
```

Comment:
"gaf" is not a compromise of the listed etymologies anymore. Let's return it to "vaf" and add fra:ouaf to the etymology.
See https://en.wiktionary.org/wiki/ouaf